inc/cachify.class.php

```diff
@@ -1694,75 +1563,68 @@ public static function validate_options($data)
+			'without_agents'   => (string) sanitize_text_field( @$data['without_agents'] ),
+			'use_apc'          => (int) $data['use_apc'],
+			'reset_on_comment' => (int) ( ! empty( $data['reset_on_comment'] )),
 		);
 	}
 
-
 	/**
-	* Display options page
-	*
-	* @since   1.0
-	* @change  2.2.2
-	*/
-
-	public static function options_page()
-	{ ?>
+	 * Display options page
+	 *
+	 * @since   1.0
+	 * @change  2.2.2
+	 */
+	public static function options_page() {
+	?>
 
 		<div class="wrap" id="cachify_settings">
 			<h1>
 				Cachify
 			</h1>
 
 			<form method="post" action="options.php">
-				<?php settings_fields('cachify') ?>
+				<?php settings_fields( 'cachify' ) ?>
 
 				<?php $options = self::_get_options() ?>
 
 				<?php /* Adds a navbar and includes the specific page */
-					if ( $options [ 'use_apc' ] !== self::METHOD_DB) {
-						$cachify_tabs = array('settings'  => esc_html__('Settings', 'cachify'),
-											'setup' => esc_html__('Setup', 'cachify'),
-									);
-
-						$current_tab = isset($_GET['cachify_tab']) ? $_GET['cachify_tab'] : 'settings';
-
-						echo '<h2 class="nav-tab-wrapper">';
-						foreach($cachify_tabs as $tab => $name){
-							$class = ($tab == $current_tab) ? ' nav-tab-active' : '';
-							$link = "?page=cachify&cachify_tab=$tab";
-							echo "<a class='nav-tab$class' href='$link'>$name</a>";
-						}
-						echo '</h2>';
-
-						switch ($current_tab){
-							case 'settings' :
-								include 'cachify.settings.php';
-								break;
-							case 'setup' :
-								if ( $options [ 'use_apc' ] === self::METHOD_HDD) {
-									if ($is_nginx) { include 'setup/cachify.hdd.nginx.php'; }
-									else { include 'setup/cachify.hdd.htaccess.php'; }
-								}
-
-								elseif ( $options [ 'use_apc' ] === self::METHOD_APC) {
-									if ($is_nginx) { include 'setup/cachify.apc.nginx.php'; }
-									else { include 'setup/cachify.apc.htaccess.php'; }
-								}
-
-								elseif ( ( $options [ 'use_apc' ] === self::METHOD_MMC) && ($is_nginx) )
-									{include 'setup/cachify.memcached.nginx.php';
-								}
-								break;
-						}
+				if ( self::METHOD_DB !== $options ['use_apc'] ) {
+					$cachify_tabs = array(
+					'settings'  => esc_html__( 'Settings', 'cachify' ),
+									'setup' => esc_html__( 'Setup', 'cachify' ),
+							);
+
+					$current_tab = isset( $_GET['cachify_tab'] ) ? $_GET['cachify_tab'] : 'settings';
+
+					echo '<h2 class="nav-tab-wrapper">';
+					foreach ( $cachify_tabs as $tab => $name ) {
+						$class = ($tab === $current_tab) ? ' nav-tab-active' : '';
+						$link = "?page=cachify&cachify_tab=$tab";
+						echo "<a class='nav-tab$class' href='$link'>$name</a>";
+					}
+					echo '</h2>';
+
+					switch ( $current_tab ) {
+						case 'settings' :
+							include 'cachify.settings.php';
+							break;
+						case 'setup' :
+							if ( self::METHOD_HDD === $options ['use_apc'] ) {
+								if ( $is_nginx ) { include 'setup/cachify.hdd.nginx.php';
```

Comment:
I just realize, we do not import `$is_nginx` and it is undefined. Would be great, if we could import it in this patch.